fairworkflows/fairworkflow.py

```diff
@@ -23,8 +22,11 @@ class FairWorkflow(RdfWrapper):
         Fair Workflows may be fetched from Nanopublications, or created through the addition of FairStep's.
     """
 
-    def __init__(self, description, uri=DEFAULT_PLAN_URI):
-        super().__init__(uri=uri)
+    def __init__(self, description, uri=None):
```

Comment:
Shouldn't we anonomyze here as well? (just like in fairstep?)

Reply:
Yes, but currently it is anonymized since we haven't loaded any 'foreign' RDF (e.g. from a nanopub) that needs anonymizing. A description is set, but that will use ```self.self_ref``` which is already a blank node.

We will indeed need to apply the anonymization here too as soon as we implement starting a FairWorkflow from existing RDF.

Reply:
Check!